frontend/src/pages/DashboardPage/QRCodeShare/QRCodeShare.css

```diff
@@ -0,0 +1,27 @@
+.share-button {
```

Comment:
Anything specific to be noted about this CSS for the QR code?

Reply:
The share button isn't using an image, rather, it is using an SVG for better clarity. THe rest of the CSS properties are quite general and are created to maintain the existing theme of the page.